examples/rpl_udp/udp.c

```diff
@@ -122,7 +122,7 @@ void udp_send(int argc, char **argv)
     memset(&sa, 0, sizeof(sa));
 
     if (address) {
-        ipv6_addr_init(&ipaddr, 0xabcd, 0x0, 0x0, 0x0, 0x0, 0x00ff, 0xfe00, (uint16_t)address);
+        ipv6_addr_init(&ipaddr, 0xabcd, 0x0, 0x0, 0x0, 0x0200, 0x00ff, 0xfe00, (uint16_t)address);
```

Comment:
but the setting of `abcd` explains a lot, why we have so many checksum errors with UDP. Why `abcd`? Even if we use the method in RFC 4944, who is saying that the PAN is `abcd` for the destination?